yt/visualization/fits_image.py

```diff
@@ -154,64 +213,159 @@ def __init__(self, data, fields=None, units=None, width=None, wcs=None):
             else:
                 self.fields.append(fd)
 
+        # Sanity checking names
+        s = set()
+        duplicates = set(f for f in self.fields if f in s or s.add(f))
+        if len(duplicates) > 0:
```

Comment:
I was a bit confused by this block. Is this renaming all duplicate instances to be "<ftype>_<fname>"?

Reply:
Never mind! I looked above to see how you were setting up `self.fields` and now I understand.